### PR TITLE
create a dedicated sa for CCM

### PR DESCRIPTION
**Pull request by zjj2wry**
**How to categorize this PR?**

/area TODO
/kind TODO
/priority normal
/platform aws

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/deployment.yaml [hunk 1]
```diff
@@ -115,6 +115,7 @@ spec:
       restartPolicy: Always
       schedulerName: default-scheduler
       terminationGracePeriodSeconds: 30
+      serviceAccountName: cloud-controller-manager
```
Comment (tennix) [inline charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/deployment.yaml:118]: What about the RBAC for cloud-controller-manager?

Comment (tennix) [reply charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/deployment.yaml:118]: Oh, I got it. The cloud-controller-manager uses static kubeconfig secret to access apiserver.

**Review by tennix — APPROVED** [review]
LGTM

**Review by aylei — APPROVED** [review]
lgtm

**Pull request merged**